.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parserOptions": {
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "extends": ["airbnb-base"],
+  "rules": {
+    "no-shadow": "off",
+    "no-param-reassign": "off",
+    "eol-last": "off",
+    "arrow-parens": "off",
+    "max-classes-per-file": ["error", 2],
+    "quotes": [
+      2,
+      "double",
+      {
+        "avoidEscape": true
+      }
+    ]
```

Comment:
As mentioned by @bolah2009, please do not change the linter config files, it's also mentioned on Pathwright, see [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png), use [these](https://github.com/microverseinc/linters-config/tree/master/javascript) linter config files that are provided by microverse.

Reply:
Whenever we save our code it places double quotes automatically which lint is picking as errors so I don’t know what you want us to do

Reply:
okay, now I get why you kept changing the rules 🙂, one way to solve this is to use `.prettierrc.js` file with the following content `module.exports = {
  singleQuote: true,
};
` but maybe there's a better way to fix this, let's see what @bolah2009 says about this

Reply:
I agree with @fatymahmed here. If prettier is your formatting tool, I think that can solve the issue you having.

Reply:
we solved the issue